rom/dev/src/wdt.rs

```diff
@@ -35,17 +33,9 @@ pub fn start_wdt(soc_ifc: &mut SocIfc) {
         if wdt_timeout_cycles == 0 {
             wdt_timeout_cycles = 1;
         }
-        cprintln!(
-            "[state] Starting the Watchdog Timer {} cycles",
-            wdt_timeout_cycles
-        );
         caliptra_common::wdt::start_wdt(
             soc_ifc,
             WdtTimeout::from(core::num::NonZeroU64::new(wdt_timeout_cycles).unwrap()),
         );
-    } else {
-        cprintln!(
-            "[state] Watchdog Timer is not started because the device is not locked for debugging"
-        );
```

Comment:
These might be worth leaving in (and shortening) since I think they help quite a bit when debugging-related test failures (which happen from time-to-time if people add code that makes boot take longer in some area)

Reply:
Sounds good. I'll revert this. This saved quite a bit of space, so the next commit may touch more places.

Reply:
@jhand2 I double checked and I think I can make it work without this change. If I need more space I'll open a separate PR.